Resolve syncing conflicts from repo_sync_working_branch to live (#1354)
MicrosoftDocs/OfficeDocs-SharePoint: OneDrive/One-Drive-Quickstart-Small-Business.md

3.  **Add OneDrive licenses.** Review your plan options in [Compare OneDrive for Business plans](https://products.office.com/onedrive-for-business/compare-onedrive-for-business-plans), and then add the licenses you need.

When you’ve completed these tasks, you’re ready to plan for, deploy, and configure the OneDrive sync app and applications. To do that, complete these three simple steps:

1.  **Plan for adoption.** For small businesses, planning for user adoption can be as simple as individually showing your users how to use OneDrive. Often, small business customers don’t consider this step for new applications, and that can negatively affect the application’s success. The section [Adopt OneDrive](#adopt-onedrive) provides helpful resources for OneDrive adoption.

> 2.  **Install and configure.** Sync app are available for the Windows and macOS operating systems that provide a seamless experience for users interacting with their files. Most small businesses start by installing the sync app on their users’ devices, and then consider the OneDrive mobile apps later. In fact, you may already have the OneDrive client on your devices. Devices running the Windows 10 operating system and devices running Windows or macOS with Microsoft Office 2016 or later will have the OneDrive sync app already. For information about how to install and configure the OneDrive sync app and mobile apps, see the section [Install and configure OneDrive](#install-and-set-up-onedrive-apps).

Reviewer: ```suggestion
2.  **Install and configure.** Sync app is available for the Windows and macOS operating systems to provide a seamless experience for users interacting with their files. Most small businesses start by installing the sync app on their users’ devices, and then consider the OneDrive mobile apps later. In fact, you may already have the OneDrive sync app on your devices. Devices running the Windows 10 operating system and devices running Windows or macOS with Microsoft Office 2016 or later will have the OneDrive sync app already. For information about how to install and configure the OneDrive sync app and mobile apps, see the section [Install and configure OneDrive](#install-and-set-up-onedrive-apps).
```

## Install and set up OneDrive apps

> You can upload, download, and interact with your OneDrive files from a web browser, but the ideal OneDrive experience comes from the Windows and Mac sync apps and the iOS and Android mobile apps. With these clients and apps, saving files to OneDrive and interacting with them is much easier than visiting a website each time you need something. Through this experience, you can seamlessly integrate OneDrive into your existing file interaction experiences.

Reviewer: ```suggestion
You can upload, download, and interact with your OneDrive files from a web browser, but the ideal OneDrive experience comes from the Windows and Mac sync apps and the iOS and Android mobile apps. With these clients and apps, saving files to OneDrive and interacting with them is much easier than visiting a website each time you need something. Through this experience, you can seamlessly integrate OneDrive into your existing file interaction experiences.
```

Most small businesses start by installing the OneDrive sync app on users’ Windows and macOS devices, and then consider the OneDrive mobile apps afterwards. You don’t need to install and configure OneDrive on all your devices before you start using it.

> [!VIDEO https://www.microsoft.com/videoplayer/embed/RE2CvJE]

### Install and configure the sync app on a Windows device

If your Windows device has either Office 2016 or Windows 10, it already has the OneDrive sync app.

For devices running older versions of Windows or on which Office 2016 is not installed, you can download the OneDrive sync app for Windows from [https://onedrive.live.com/about/download](https://onedrive.live.com/about/download/).

> [!NOTE]
> If the device has an older version of the sync app, you’ll be asked to uninstall it when you install the new one.

> Configuring OneDrive for Windows is simple, but if you want to see a demonstration, see [Sync files with the OneDrive sync app in Windows](https://support.office.com/article/615391c4-2bd3-4aae-a42a-858262e42a49) 

Reviewer: Please  check - the link takes you to the article "Sync files with the OneDrive sync **client** in Windows". Please either remove the reference or change the link.

### Install and configure OneDrive on a macOS device

> To install the OneDrive sync app on a computer running macOS, just follow the steps in [Sync files with the OneDrive sync app on Mac OS X](https://support.office.com/article/d11b9f29-00bb-4172-be39-997da46f913f). The setup experience is similar to that for Windows. For more information about OneDrive on macOS, see [OneDrive for Mac – FAQ](https://support.office.com/article/3fc4062c-8051-4392-bff1-551e32840cd0). 

Reviewer: Again please check - the link goes to the article "Sync files with the OneDrive sync **client** on Mac OS X".
